ui/src/views/offering/AddNetworkOffering.vue

```diff
@@ -237,7 +237,7 @@
             </a-radio-button>
           </a-radio-group>
         </a-form-item>
-        <a-form-item>
+        <a-form-item name="serviceofferingid" ref="serviceofferingid">
```

Comment:
@weizhouapache , this line caused a conflict during forward merging of 4.17 after merging #7057. Can you have a look if I made the right call here?